package.json

```diff
@@ -17,10 +17,7 @@
     "xkp": "cli.js",
     "xkpasswd": "cli.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vot/xkpasswd-node.git"
-  },
+  "repository": "https://github.com/vot/xkpasswd-node.git",
```

Comment:
Hmm cool. I'd rather just go with the documented npm way, just for the sake of ensuring it doesn't cause any weird `package.json` issues in older versions or private/cache servers.